src/app/components/ColorSelector/ColorSelector.module.css

```diff
@@ -24,8 +24,11 @@
   list-style: none;
   display: grid;
   grid-template-columns: repeat(5, auto);
-  padding: 0;
   grid-row-gap: 1em;
+  border: 1px solid #707070;
```

Comment:
Is px a good unit for this purpose? 
Maybe you want to use CSS variable names for the color? :)